session/tidb_test.go

```diff
@@ -34,6 +34,10 @@ import (
 	"github.com/pingcap/tidb/util/testleak"
 )
 
+func init() {
+	runningUnitTest = true
```

Comment:
`TestSQLMode` in executor package wants to set this variable also, can we export it here?

Reply:
many packages need this variable to accelerate CI 😂

Reply:
Done